lib/cooked_post_processor.rb

```diff
@@ -361,8 +361,6 @@ def pull_hotlinked_images(bypass_bump = false)
     return if disable_if_low_on_disk_space
     # don't download remote images for posts that are more than n days old
     return unless @post.created_at > (Date.today - SiteSetting.download_remote_images_max_days_old)
-    # we only want to run the job whenever it's changed by a user
-    return if @post.last_editor_id == Discourse.system_user.id
```

Comment:
Why was this removed?

Reply:
After the system_user edit for `downloaded local copies of images` again we need to call `pull_hotlinked_images` to replace broken & large images with placeholder.

In 1st step downloaded local images will replaced in `raw` content. Here post will be updated automatically by system user. Then `CookedPostProcess` will be called and downloaded onebox thumbnail will replaced with local copy. Now we have to call `pull_hotlinked_images` to check broken and large image (we can't do it in 1st step since there raw post only changed. Now we changing cooked content). Now last edit is made by system user. That's why I removed that.

1. `pull_hotlinked_images`
2. `CookedPostProcess`
3. `pull_hotlinked_images`

Reply:
@ZogStriP I just removed it from this PR to be it merged. I can add the above code in another PR. But it gives spec error since [this code](https://github.com/discourse/discourse/blob/master/spec/components/cooked_post_processor_spec.rb#L550). Any help?